pom.xml

```diff
@@ -68,6 +68,12 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-spring-boot-starter</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.vaadin.hilla</groupId>
+                    <artifactId>hilla-react</artifactId>
```

Comment:
This shouldn't be needed after https://github.com/vaadin/hilla/pull/2015, as Flow automatically adds or doesn't add the generator deps.